server/data/incidentTypeConfiguration/formWizard.ts

```diff
@@ -1,5 +1,5 @@
 import FormWizard from 'hmpo-form-wizard'
-import { IncidentTypeConfiguration } from './types'
+import type { IncidentTypeConfiguration } from './types'
```

Comment:
👌

Reply:
sorry, i kno these keywords are trivial and make no difference to the output but i cannot help myself 😅